jupyter/src/incremental_learning/transforms.py

```diff
@@ -65,12 +65,13 @@ def partition_on_metric_ranges(seed : int,
     '''
 
     if matches_columns(metric_features, dataset):
-        print(list(dataset.columns), 'does not contain', metric_features)
+        print('ERROR:', list(dataset.columns), 'does not contain', metric_features)
```

Comment:
It would be better if you use the `logger` from `config` instead of printing the messages:
```python
from .config import logger
...
logger.error("{} does not contain {}".format(dataset.columns, metric_features))
```

Same below.

Reply:
Corrected in [1b78fa7](https://github.com/elastic/ml-cpp/pull/2021/commits/1b78fa701cd44cc288e22992683f762b40718c3f).